src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -17,6 +17,7 @@
     <!-- SYSTEM_NET_PRIMITIVES_DLL is required to allow source-level code sharing for types defined within the
          System.Net.Internals namespace. -->
     <DefineConstants>$(DefineConstants);SYSTEM_NET_PRIMITIVES_DLL</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
If you change the TargetGroup=uap then you should also update the define to be uap.